src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -947,6 +1074,7 @@ private void ParseHelper(string[] args)
                 }
                 else if (MatchSwitch(switchKey, "mta", "mta"))
                 {
+                    ParametersUsed |= ParameterBitmap.MTA;
```

Comment:
Same here, this line should be moved to `_staMode = false;` to keep consistent with the rest.